README.md

```diff
@@ -1 +1,46 @@
-# ToDo-Review
\ No newline at end of file
+# Today's To Do
+
+## This is a simple daily todo application
+
+## Getting Started
+
+1. copy the Url from my github account
+2. clone the below url to your local machine
+3. Run git clone git@github.com:ismailakinkunmi/portfolio-site.git
+
+## Technologies Used
+
+1. Eslint
+2. Webpack
+3. HTML5
+4. CSS#
+5. JavaScript
```

Comment:
Why have you repeated these languages two times? This is considered as Repeating yourself. Please I humbly suggest that you correct that.